src/Compilers/CSharp/Portable/Compilation/CSharpDiagnosticFilter.cs

```diff
@@ -82,8 +90,20 @@ public static Diagnostic Filter(Diagnostic d, int warningLevelOption, ReportDiag
 
         // Take a warning and return the final deposition of the given warning,
         // based on both command line options and pragmas.
-        internal static ReportDiagnostic GetDiagnosticReport(DiagnosticSeverity severity, bool isEnabledByDefault, string id, int diagnosticWarningLevel, Location location, string category, int warningLevelOption, ReportDiagnostic generalDiagnosticOption, IDictionary<string, ReportDiagnostic> specificDiagnosticOptions)
+        internal static ReportDiagnostic GetDiagnosticReport(
+            DiagnosticSeverity severity,
+            bool isEnabledByDefault,
+            string id,
+            int diagnosticWarningLevel,
+            Location location,
+            string category,
+            int warningLevelOption,
+            ReportDiagnostic generalDiagnosticOption,
+            IDictionary<string, ReportDiagnostic> specificDiagnosticOptions,
+            out bool hasPragmaSuppression)
```

Comment:
So suppressed diagnostics will be displayed in the error list by default? How will the user know that the diagnostic has been suppressed?